### PR TITLE
Add Yamashita proposal implementation

### PR DESCRIPTION
**Pull request by sporejack**
# Issue
#7 

# Implementation
[DAOCheck](https://github.com/mycelium-ethereum/DAOCheck) was provided with the proposal plan specified in the proposal implementation text added to the repository by this PR.

Manual inspection was performed throughout the process.

## Things to Check ##
 - [x] Targets are the same
 - [x] Targets are correct
 - [x] Quantity of TCR is correct (i.e., `amount`)
 - [x] TCR address is correct (i.e., `asset`)
 - [x] Cancellable flags are distinct (i.e., `isFixed1 XOR isFixed2 == true`)
 - [x] Quantity of TCR transferred in the first call is the **exact** sum of the quantities of both vesting schedules
 - [x] Initial transfer of TCR goes to the correct destination (i.e., the (correct) vesting contract)

### CODE REVIEW

**Comment by sporejack** [comment]
@BenjaminPatch discovered an issue with the initial payload data in that the second call to `vest` (i.e., the **last** call of the entire proposal plan) had `isFixed` set to `true`, when it should have been `false`. This is due to a bug in DAOCheck which we're investigating currently.

I've applied a manual fix in fdee1e0 (please read the commit message there).

File: proposals/14-yamashita-proposal.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+TARGET:
+    0x9C4A4204B79dd291D6b6571C5BE8BbcD0622F050 (https://etherscan.io/address/0x9C4A4204B79dd291D6b6571C5BE8BbcD0622F050)
+DATA:
+    0xa9059cbb000000000000000000000000707b6be09028e78d2a667db7596b2803c112f9b2000000000000000000000000000000000000000000084595161401484a000000
+
+============ Call 1 ============
+TARGET:
+    0x707b6be09028e78d2a667db7596b2803c112f9b2 (https://etherscan.io/address/0x707b6be09028e78d2a667db7596b2803c112f9b2)
+DATA:
+    0x0d53588300000000000000000000000087c2f090997b0e158c10399605d19c41ec47b49b0000000000000000000000000000000000000000000422ca8b0a00a4250000000000000000000000000000009c4a4204b79dd291d6b6571c5be8bbcd0622f05000000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000005b00000000000000000000000000000000000000000000000000000000610b8c70
```
Comment (BenjaminPatch) [inline proposals/14-yamashita-proposal.md:139]: I might be missing something, but I believe this is still not correct. It corresponds to the proposal data of a vesting contract which has `isFixed` set to `true`, but `isFixed` is `false` for call one (according to the data given starting on line 42)

**Review by BenjaminPatch — CHANGES_REQUESTED** [review]

File: proposals/14-yamashita-proposal.md (hunk 1 above)
Comment (sporejack) [reply proposals/14-yamashita-proposal.md:139]: Missed this in fdee1e0; consult 387916a.

**Comment by Paul4912** [comment]
Im reading the proposal and it says "starting on the day that this proposal is executed." Isn't the timestamp you put in the date in the past now?

**Review by raymogg — APPROVED** [review]
Confirmed
- targets are correct
- amounts are correct
- vesting params are correct
- start date (unix timestamp) is correct
- duration is correct
- function signatures are correct
- function call data is correct

Note:
In the order of function calls passed into the generation code, `isFixed` is set to false in call 2 and true in call 3, however in the payload data these are reversed.

**Comment by raymogg** [comment]
> e

Yeah moving forward we will get proposers to clear up this wording. We are considering proposals passed as when the snapshot vote has passed. This means time between approval and on chain execution does not affect vesting schedules.

**Comment by raymogg** [comment]
> Im reading the proposal and it says "starting on the day that this proposal is executed." Isn't the timestamp you put in the date in the past now?

Yeah moving forward we will get proposers to clear up this wording. We are considering proposals passed as when the snapshot vote has passed. This means time between approval and on chain execution does not affect vesting schedules.

**Comment by sporejack** [comment]
> Im reading the proposal and it says "starting on the day that this proposal is executed." Isn't the timestamp you put in the date in the past now?

Yes, because the [vote passed in the past](https://vote.tracer.finance/#/tracer.eth/proposal/QmXJXyfgcJoDXEuHxuZoGT5AqVTpwyBma73kCdfr72t6pf).

**Comment by sporejack** [comment]
> Note:
> In the order of function calls passed into the generation code, `isFixed` is set to false in call 2 and true in call 3, however in the payload data these are reversed.

Consult 55844c3.

**Review by raymogg — APPROVED** [review]

**Review by mynameuhh — APPROVED** [review]

**Review by ScaredIbis — APPROVED** [review]

**Pull request merged**